lib/portlayer/exec/container.go

```diff
@@ -220,3 +221,27 @@ func (c *Container) Stop(ctx context.Context) error {
 
 	return nil
 }
+
+func (c *Container) VmRemove(ctx context.Context) error {
+	containersLock.Lock()
+	c.Lock()
+	defer containersLock.Lock()
```

Comment:
The `defer` keryword calls the deferred item in LIFO order.  Disregard my comment above.  TIL!!